recipes/qcs-api-client-common/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "qcs-api-client-common" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: qcs-api-client-common-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/rigetti/qcs-api-client-rust/archive/refs/tags/common/v{{ version }}.tar.gz
+  sha256: ee84c504bd5843ad8f129b50594ab9afaaa8bfa32d8f43a016b4ecc5bce1ae37
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - maturin >=1.0.0,<2.0.0
```

Comment:
maturin should be in host and only additionally in build for cross-compiles, similar to python.